docs/releasing.md

```diff
@@ -0,0 +1,88 @@
+# Releasing - For maintainers only
+
+This document explains how to publish all opentelemetry-operations-js Node
+modules at version x.y.z. Ensure that you’re following semver when choosing a
+version number.
+
+- [Example PR #139](https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/pull/139)
+- [Example Tag v0.4.0](https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/releases/tag/v0.4.0)
+
+## Update to latest locally
+
+Use `git fetch` and `git checkout origin/master` to ensure you’re on the latest commit. Make sure
+you have no unstaged changes. Ideally, also use `git clean -dfx` to remove all ignored and
+untracked files.
+
+## Create a new branch
+
+Create a new branch called `x.y.z-proposal` from the current commit.
+
+## Use Lerna to prepare each package for release
+
+Use below command to initialize all package directories.
+
+```bash
+npm install
+```
+
+Also, use `lerna publish --skip-npm --skip-git` to bump version numbers.
+Ensure that the version being bumped to is `x.y.z`. This should create some unstaged changes.
+
+## Create a new commit
+
+Create a new commit with the exact title: `chore: x.y.z release proposal`.
+
+## Use the Changelog to create a GitHub Release
+
+On [GitHub
+Releases](https://github.com/GoogleCloudPlatform/opentelemetry-operations-js/releases),
+start creating a release and put a changelog generated by git log in the body
+(TODO: keep a proper changelog):
+
+```bash
+git log --oneline --no-decorate <previous release tag>..HEAD
+```
+
+Save it as a draft, don’t publish it. Don’t forget the tag -- call it `vx.y.z`
+and leave it pointing at `master` for now (this can be changed as long as the
+GitHub release isn’t published).
+
+## Create a new PR
+
+Push the branch to GitHub and create a new PR with that exact name. The commit body should just
+be a link to the *draft* notes. Someone who can access draft notes should approve it, looking
+in particular for test passing, and whether the draft notes are satisfactory.
+
+## Merge and pull
+
+Merge the PR, and pull the changes locally (using the commands in the first step). Ensure that
+`chore: x.y.z release proposal` is the most recent commit.
+
+## Publish all packages
+
+Publish with Wombot (Google).
+
+Go into each directory and use `npm publish` to publish the package. You can use the following
+script to automate this.
+
+```bash
+#!/bin/bash
+
+for package in $(ls packages/**/package.json); do
+  pushd $(dirname $package)
+  npm publish
```

Comment:
Would you like to add lines about wombat token generation and all?

Reply:
Discussed offline